packer/bootstrap.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+echo "Updating dnf packages"
+sudo dnf update -y
+
+echo "Installing kernel-devel for vboxadditions support"
+sudo dnf install kernel-devel -y
+
+echo "Installing Python and Ansible"
+sudo dnf install -y python3 python3-pip python3-setuptools python3-wheel
+
+sudo python3 -m pip install --upgrade pip
+pip3 install ansible --user
+
+echo "Installing guest additions"
+mkdir /tmp/isomount
+sudo mount ~/VBoxGuestAdditions.iso /tmp/isomount
+sudo /tmp/isomount/VBoxLinuxAdditions.run
+sudo umount /tmp/isomount
+sudo rm -rf /tmp/isomount
+
+echo "Adding our user to the virtualbox group"
+sudo usermod -a -G vboxsf "$USER"
+
+echo "Installing minimal gnome"
+sudo dnf groupinstall "X Window System" -y
+sudo dnf install gnome-classic-session gnome-terminal nautilus-open-terminal control-center liberation-mono-fonts -y
+sudo unlink /etc/systemd/system/default.target
+sudo ln -sf /lib/systemd/system/graphical.target /etc/systemd/system/default.target
+
+echo "Dropping Notes on Desktop"
```

Comment:
in the ddh version i  have code to autorun this file as a gnote
cant decide if I love it or hate it :D

Reply:
I'm definitely down with it either way- the only catch is that they have to have done the `.env` copy first, so may require re-organizing things in the docs a little